editor/components/block-list/block.js

```diff
@@ -538,12 +596,18 @@ export class BlockListBlock extends Component {
 						layout={ layout }
 						isFirst={ isFirst }
 						isLast={ isLast }
+						draggable={ true }
+						onDragStart={ this.onDragStart }
```

Comment:
Do these events need to be applied to every one of the child components? Can't we add a single drag event to the root wrapper component? If it's an issue of competing with `onMouseDown`... well, I don't like that we use `onMouseDown`, so maybe there's a refactoring opportunity there.

Reply:
Arghh, I wish there was an easier way to accomplish this whole effect. The way I have designed the logic to handle dragging is we need to apply these methods to everything that we want to use as a drag handle (a point that the user can grab something from). They will behave the same way regardless of where they are applied. This way we don't have to experiment with an overlay that will interfere with other drag logic (i.e. resizing an image, selecting text, or anything else).

So to answer your question: these events (returned from the closures in `withDragging`) need to be applied to every area of the UI that we want to use as a drag handle. Irrespective of where it is.